cobc/pplex.l

```diff
@@ -106,6 +107,9 @@ static int ppwrap (void) {
 
 #define PLEX_COND_DEPTH		16
 
+// replace yytext by some simplified text in ppecho()
```

Comment:
please change that to c89 comment and explain why we do that - it replaces all texts below ?!?